networking/network_security/configuring-ipsec-ovn.adoc

```diff
@@ -13,6 +13,8 @@ IPsec is disabled by default. It can be enabled either during or after installin
 [IMPORTANT]
 ====
 If your cluster uses link:https://www.redhat.com/en/topics/containers/what-are-hosted-control-planes[{hcp}] for Red Hat {product-title}, IPsec is not supported for IPsec encryption of either pod-to-pod or traffic to external hosts.
+
+You must disable IPsec before updating to {product-title} 4.15. There is a known issue that can cause interruptions in pod-to-pod communication if you update without disabling IPsec.
```

Comment:
please add link to the known issue: https://issues.redhat.com/browse/OCPBUGS-41823

Reply:
@pperiyasamy  PTAL if it's accurate to say pod-to-pod communication will be interrupted.

Reply:
I can't link to a bug that is RH security. Please provide CCFR in this bug: https://issues.redhat.com/browse/OCPBUGS-43323. I will rewrite it and then link to it.

Reply:
what is CCFR?

Reply:
cause, consequence, fix, and result

Reply:
the content looks ok to me.